managed/src/main/java/com/yugabyte/yw/models/Backup.java

```diff
@@ -210,7 +210,8 @@ public static Backup fetchByTaskUUID(UUID taskUUID) {
       Customer customer = Customer.get(customerUUID);
       Set<UUID> allUniverseUUIDs = Universe.getAllUUIDs(customer);
       List<Backup> backupsWithValidUniv = backups.stream()
-        .filter(backup -> allUniverseUUIDs.contains(backup.getBackupInfo().universeUUID))
+        .filter(backup -> allUniverseUUIDs.contains(backup.getBackupInfo().universeUUID) && 
+            !Universe.get(backup.getBackupInfo().universeUUID).getUniverseDetails().universePaused)
```

Comment:
Given that we are allowing deletion of Universe without deleting backups; as discussed before this check should be:
**Universe not present || Universe not paused**

So remove the allUniverseUUIDs check. 

Also `Universe.get` throws exception when not found. This has been a source of a bug. I am replacing it to Univers.getOrBadRequest and Universe.maybeGet  methods so this does not come as a surprise. 

Meanwhile you should add a static method to Universe:
```
    static boolean isPaused(uuid) {
        return true if universe exists && is paused;
        else return false;
    }
```

Then you can filter on ```!Universe.isPaused(universeUUID)```